Doc/library/typing.rst

```diff
@@ -2392,6 +2392,13 @@ types.
          >>> Point3D.__total__
          True
 
+      This attribute reflects *only* the value of the ``total`` argument
+      to the current ``TypedDict`` class, not whether the class is semantically
+      total. For example, a ``TypedDict`` with ``__total__`` set to True may
```

Comment:
Just to bring it in line with what other parts of the typing docs convention
```suggestion
      total. For example, a ``TypedDict`` with ``__total__`` set to ``True`` may
```

Reply:
This change is a backport. If you want to fix the doc, you should fix it in the main branch first.